CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0-beta.4] - 2020-02-17
+
+## [3.1.0-beta.3] - 2020-02-16
+
+## [3.1.0-beta.2] - 2020-02-16
```

Comment:
Beta releases shouldn't be made on the branch that will be merged into master...